docs/streams/developer-guide/processor-api.html

```diff
@@ -343,6 +344,38 @@ <h2>
 </pre></div>
                 </div>
             </div>
+            <div class="section" id="timestamped-state-stores">
+                <span id="streams-developer-guide-state-store-timestamps"></span><h3><a class="toc-backref" href="#id11">Timestamped State Stores</a><a class="headerlink" href="#timestamped-state-stores" title="Permalink to this headline"></a></h3>
+                <p>
+                    Starting in Kafka Streams 2.3, you can store record timestamps in KTables.
```

Comment:
> Starting in Kafka Streams 2.3

Do we need this?

> can store

`KTables` always store timestamps by default. Should we rephrase?